Signal/translations/ar.lproj/Localizable.strings

```diff
@@ -1873,9 +1873,6 @@
 /* Label for the block list section of the settings view */
 "SETTINGS_BLOCK_LIST_TITLE" = "تم الحظر";
 
-/* Accessibility hint for the settings button */
```

Comment:
I agree with removing this redundant string, and thanks for trying to keep things tidy, but just an FYI for how we manage localizations:

1. Strings/comments are added and removed from the source code as needed using the NSLocalizedString macros.

2. run `Signal/translations/bin/auto-genstrings` to extract translations into en.lproj and remove stale entries from en.lproj. (never add or remove entries directly)

3. Never edit any translations/*.lproj/* files except English. The other localization files are only edited by Signal staff as a result of our release management scripts.

So, just to keep our process orderly, could you remove these changes from the l10n files?

I've added some notes to our [Translations Readme](https://github.com/signalapp/Signal-iOS/blob/master/Signal/translations/TRANSLATIONS.md)

Reply:
👍🏽 I'll modify the patch.